packages/xchain-binance/__tests__/client.test.ts

```diff
@@ -650,4 +650,19 @@ describe('BinanceClient Test', () => {
     bnbClient.setNetwork(Network.Testnet)
     expect(bnbClient.getExplorerTxUrl('testTxHere')).toEqual('https://testnet-explorer.binance.org/tx/testTxHere')
   })
+
+  it('should broadcast a deposit to thorchain inbound address', async () => {
+    bnbClient.setNetwork(Network.Testnet)
+    const txHash = await bnbClient.deposit({
+      asset: {
+        chain: Chain.Binance,
+        synth: false,
+        symbol: 'BUSD-74E',
+        ticker: 'BUSD',
+      },
+      amount: assetToBase(assetAmount(1)),
+      memo: '=:THOR.RUNE:tthor19kacmmyuf2ysyvq3t9nrl9495l5cvktj5c4eh4',
+    })
+    expect(txHash).toEqual(expect.any(String))
```

Comment:
`toEqual(expect.any(String))` -> `toEqual(tx-mockid)` (or whatever in mocks is defined)